tools/xml/csv2xml.py

```diff
@@ -15,7 +15,7 @@
 #   may be used to endorse or promote products derived from this work without
 #   specific prior written permission.
 #
-# THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
```

Comment:
I just check the license in templates, which use `""`. Not sure where you get this update.